src/main/java/com/opentable/db/postgres/embedded/EmbeddedPostgreSQL.java

```diff
@@ -376,7 +376,6 @@ public static EmbeddedPostgreSQL start() throws IOException
         Builder() {
             config.put("timezone", "UTC");
             config.put("synchronous_commit", "off");
-            config.put("checkpoint_segments", "64");
```

Comment:
Is there an equivalent option?  Should we set something else instead?  What functionality are we losing here?